tests/tests_lvm_pool_members.yml

```diff
@@ -44,6 +44,12 @@
                     ansible_facts.distribution_major_version =='8' }}"
         is_fedora: "{{ ansible_facts.distribution == 'Fedora' }}"
 
+    # end early when running with old blivet where removing PVs from a VG fails
+    - meta: end_play
+      when: ((is_fedora and blivet_pkg_version is version("3.4.3-1", "<")) or
+             (is_rhel8 and blivet_pkg_version is version("3.4.0-10", "<")) or
+             (is_rhel9 and blivet_pkg_version is version("3.4.0-14", "<")))
```

Comment:
The EL7 tests are failing here too.  do we need a `is_rhel7` setting as well, and a check for `is_rhel7 and blivet_pkg_version is xxx`?

Reply:
I'm testing a fix now